lib/tpm2_capability.c

```diff
@@ -33,38 +33,97 @@
 
 #include <tss2/tss2_sys.h>
 
+#include <string.h>
+
 #include "log.h"
 #include "tpm2_util.h"
 
 #include "tpm2_capability.h"
 
+static int tpm2_get_property_data_size(TPM2_CAP capability) {
+    int size;
```

Comment:
size_t or unsigned? I don't see why this needs to be signed.

single line of whitespace between function and body.

```C
static size_t tpm2_get_property_data_size(...) {

    size_t size;
...